examples/admin_policy/example_server/policy_server.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+"""Example RESTful admin policy server for SkyPilot."""
+
+import argparse
+
+import example_policy
+from fastapi import FastAPI
+from fastapi import Request
+from fastapi.responses import JSONResponse
+import uvicorn
+
+import sky
+
+app = FastAPI(title="Example Admin Policy Server", version="1.0.0")
+
+
+@app.post('/')
```

Comment:
Can we have one more policy in the same file as well, e.g. using the one of the example policies we already have? The two policies could be on the endpoint `/do_nothing_policy` and `/autostop_policy`. 

On the client side (the readme), we can have the user specify `admin_policy: http://localhost:8080/autostop_policy` to showcase, how a specific policy could be specified.

Reply:
Sounds great!

Reply:
Offline discussed: will put it to another PR